src/main/java/com/playmonumenta/networkchat/NetworkChatProperties.java

```diff
@@ -72,6 +77,10 @@ private void loadInternal(Plugin plugin, @Nullable CommandSender sender) {
 			mChatCommandDeleteEnabled = config.getBoolean("ChatCommandDelete", mChatCommandDeleteEnabled);
 		}
 
+		if (config.isBoolean("ChatCommandDescription")) {
+			mChatCommandDescriptionEnabled = config.getBoolean("ChatCommandDescription", mChatCommandDescriptionEnabled);
```

Comment:
Reminder to update the build shard's config so that this is disabled there; best to send another reminder on Discord before update.